server/routerlicious/.changeset/silly-papayas-hammer.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/server-lambdas": major
+"@fluidframework/server-routerlicious": major
+"@fluidframework/server-routerlicious-base": major
+"@fluidframework/server-services-shared": major
```

Comment:
I think this specific changeset can remove `services-shared`, because that is covered in the other one. Additionally, `lambdas` only has the change that `"disposed"` event is being emitted by DeliLambdaFactory. That would be a separate `minor` changeset.

It would also be good, IMO, to indicate in the changeset description that the reason for `major` (i.e. breaking change) is that there is a new required param, `redisClientConnectionManagers`, in the Nexus, Alfred, and Riddler RunnerFactories. This is scoped to `r11s-base`.

Lastly, the `r11s` change could be included along with the `lambdas` changeset because those 2 are closely coupled, but separate from the r11s-base changes